scripts/travis_install

```diff
@@ -15,12 +15,12 @@ rm -rf $MINICONDA.sh
 python -V
 
 PINNED_PKGS=$(cat <<EOF
-binstar ==0.10.1
+
 EOF
 )
 echo -e "$PINNED_PKGS" > /home/travis/miniconda/conda-meta/pinned
 
-DEPS_TRAVIS="python=$TRAVIS_PYTHON_VERSION conda-build jinja2"
+DEPS_TRAVIS="python=$TRAVIS_PYTHON_VERSION conda-build jinja2 anaconda-client"
```

Comment:
`anaconda-client` will be used when we upload packages. in the past, I installed `binstar` here: https://github.com/bokeh/bokeh/blob/master/.travis.yml#L55
Let's put the anaconda-client in the same place... they are "upload" dependencies, so I would like to have it separated... what do you think?